### ISSUE TITLE
Cycript choose()

### ISSUE THREAD

**Issue by r3ggi** [nowsecure/frida-cycript#11]
Do you plan to implement choose() function?

In my case when use it returns
```
cy# choose(UIView)
throw new ReferenceError("identifier 'choose' undefined")
```

**Comment by oleavr** [comment]
Definitely. It should only be a matter of implementing this on top of Frida's `ObjC.choose()`.

PR welcome!

**Comment by dharmeshbaskaran** [comment]
I find the same issue. Is this fixed?

**Comment by oleavr** [comment]
@dharmeshbaskaran Nope, still waiting for somebody to contribute this feature.

**Comment by oleavr** [comment]
Implemented in #28.
